airbyte-integrations/connectors/source-shopify/metadata.yaml

```diff
@@ -17,6 +17,7 @@ data:
   githubIssueLabel: source-shopify
   icon: shopify.svg
   license: ELv2
+  maxSecondsBetweenMessages: 60
```

Comment:
Checking shopify, which rate limit is by minute? I only see some per second

Reply:
You're right. I didn't double check the ones not included originally. I'll change it to 1s, and FB to 300s.